src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -135,6 +140,14 @@ public void parse_optionalFieldsMissing_success() {
         Person expectedPerson = new PersonBuilder(AMY).withTags().build();
         assertParseSuccess(parser, NAME_DESC_AMY + PHONE_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY,
                 new AddCommand(expectedPerson));
+
+        // blank email address
+        Person expectedPersonBlankEmail = new PersonBuilder(ERIC).withTags().build();
+        assertParseSuccess(parser, NAME_DESC_ERIC + PHONE_DESC_ERIC + EMAIL_DESC_ERIC
+                + ADDRESS_DESC_ERIC, new AddCommand(expectedPersonBlankEmail));
+        // no email address tag
```

Comment:
minor nit but maybe you could leave a line between 147 and 148, to be consistent with the other parts of the code?

Reply:
Oo lowkey agree -- might look neater